c2corg_ui/format/ltag.py

```diff
@@ -0,0 +1,64 @@
+"""
+LTag Extension for Python-Markdown
+====================================
+
+See https://regex101.com/r/g2Zjqn/2/ for regex tests
+
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from markdown.extensions import Extension
+from markdown.blockprocessors import BlockProcessor
+from markdown.util import etree
+import re
+
+
+class LTagProcessor(BlockProcessor):
+    """ Process Tables. """
+
+    RE_LTAG_BLOCK = re.compile(r'^(?:L#(?:[^|\n]*(?:\|[^|\n]*)+)\n?)+$')
+
+    def __init__(self, parser):
+        super(LTagProcessor, self).__init__(parser)
+
+    def test(self, parent, block):
+        is_table = self.RE_LTAG_BLOCK.search(block) is not None
+
+        return is_table
+
+    def run(self, parent, blocks):
+
+        block = blocks.pop(0)
+        row_count = 1
+
+        if not self.test(parent, block):
+            return
+
+        rows = block.split('\n')
+        table = etree.SubElement(parent, 'table')
+        table.set('class', 'ltag')
+        tbody = etree.SubElement(table, 'tbody')
+        for row in rows:
+            tr = etree.SubElement(tbody, 'tr')
+            cols = row.split('|')
+            for col in cols:
+                td = etree.SubElement(tr, 'td')
+                td.text = col.strip().replace('L#', "L" + str(row_count))
```

Comment:
FYI, I've bolded first cell using this : 
https://github.com/cbeauchesne/campui/blob/master/app/static/campui/scss/campui.scss#L172

It avoid using repetitive class in generated code.

Although, I would recommend to not parse table if you are not sure that they respect the part of the spec you have coded. Otherwise, you would introduce false interpretations that mde most of topo false, and, worst, new topo will respect the newly interpretation, and it will be very difficult to correct everything.